glob/CMakeLists.txt

```diff
@@ -201,12 +201,12 @@ new_ec_test(braces_numeric_range7 braces.in 121 "^[ \t\n\r]*$")
 new_ec_test(braces_numeric_range8 braces.in 060 "^[ \t\n\r]*$")
 
 # alphabetical brace range
-new_ec_test(braces_alpha_range1 braces.in {ardvark..antimater} "^words=a[ \t\n\r]*$")
-new_ec_test(braces_alpha_range1 braces.in a "^[ \t\n\r]*$")
+new_ec_test(braces_alpha_range1 braces.in {aardvark..antimater} "^words=a[ \t\n\r]*$")
```

Comment:
Did you intend to change this line?

Reply:
Yes, since "aardvark" is spelled correctly on line 206 I assumed it should have been spelled correctly here as well.